sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/utils/FileUtils.java

```diff
@@ -192,6 +196,25 @@ public boolean accept(File dir, String name) {
         });
     }
 
+    /**
+     * read file and convert it to string
+     *
+     * @param filePath file path
+     * @return String
+     */
+    public static String readFileToString(String filePath) {
+        // Read all bytes from the file at once
+        byte[] bytes = new byte[0];
```

Comment:
bytes does not need to be instantiated and will be overwritten later.

Reply:
done